2022-08-trading-orders.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The basics of trading - The orders
+subtitle: 
+slug: trading-orders
+tags: crypto, trading, strategy, automation, orders, blog
+cover: https://images.unsplash.com/photo-1642751226315-e6dc6b47fd54?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3774&q=80
+domain: blog.octobot.online
+--- 
+
+Trading cryptocurrencies can be exciting, but also take a lot of time if you don’t know how to do it. So, there are many different types of orders you can make to buy or sell on your exchange  according to what you want. Read our introductory article and get a basic understanding of the orders available.
+
+# Market order, the easiest
+
+When you place a market order, you're telling your broker to buy or sell shares at the best available price. If you're buying, your broker will try to get the lowest possible price. If you're selling, they'll try to get the highest. 
```

Comment:
```suggestion
When you place a market order, you're telling your broker (the exchange) to buy or sell shares or crypto-assets at the best available price. If you're buying, your broker will try to get the lowest possible price. If you're selling, it will try to get the highest. 
```